src/server/node_manager.c

```diff
@@ -6581,10 +6581,10 @@ set_nodes(void *pobj, int objtype, char *execvnod_in, char **execvnod_out, char
 			    (pnode = find_alien_node(vname)) == NULL) {
 				if (svr_init && (pjob->ji_qs.ji_svrflags & JOB_SVFLG_RescUpdt_Rqd))
 					continue;
+				log_eventf(PBSEVENT_DEBUG, PBS_EVENTCLASS_SERVER, LOG_INFO,
+					   __func__, "Unknown node %s received", vname);
```

Comment:
Knowing the node name is good, but since this happens while parsing the select spec for a job, I think you should keep the class as PBS_EVENTCLASS_JOB and add the jobid back because the error has occurred wrt running a job.